python/agents/auto-insurance-agent/README.md

```diff
@@ -0,0 +1,182 @@
+# Auto insurance Agent using Apigee API hub
+
+## Overview
+
+This agent sample acts as a helpful virtual assistant for an auto insurance company. It is able to register new members and perform several functions for existing members, including filing claims, providing roadside assistance, and returning information about reward offers from partner companies.  The agent uses several tools to fulfill these tasks. The tools are custom APIs registered in [Apigee API hub](https://cloud.google.com/apigee/docs/apihub/what-is-api-hub).
+
+## Agent Details
+
+| Attribute | Detail |
+|---|---|
+|   Interaction Type |   Conversational |
+|   Complexity |   Easy |
+|   Agent Type |   Multi Agent |
+|   Components |   Tools, ApiHubToolset |
+|   Vertical |   Financial Services |
+
+### Agent Architecture
+
+![architecture](arch.png)
+
+### Key Features
+
+#### Tools
+
+The agent has access to the following tools:
+
+- `membership`: Registers new members.
+- `claims`: Handles claims.
+- `roadsideAssistance`: Provides roadside assistance, including towing services.
+- `rewards`: Finds nearby reward offers from partners.
+
+The tools are provided by custom APIs. The specifications are imported to [API hub](https://cloud.google.com/apigee/docs/apihub/what-is-api-hub) and then referenced in the agent code using ADK's built-in [ApiHubToolset](https://google.github.io/adk-docs/tools/google-cloud-tools/#apigee-api-hub-tools). This lets agent developers easily turn any existing API from their organization's API catalog into a tool with just a few lines of code. For this sample, the APIs themselves are served using [Apigee](https://cloud.google.com/apigee).
+
+## Setup and Installation
+
+### Prerequisites
+
+- Python 3.12+
+-   Poetry for dependency management and packaging
+    -   See the official
+        [Poetry website](https://python-poetry.org/docs/) for more information. To install Poetry run:
+    ```bash
+    pip install poetry
+    ```
+- Google Cloud Project with the following roles assigned
```

Comment:
Please also add roles required to view logs during deployment.

Deployment output gives the following:
```
View progress and logs at https://console.cloud.google.com/logs/query?project=apigee-ai
```

and I was unable to access the logs;